ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -253,6 +269,22 @@ export default class SelectHardware extends Component {
         {steps.map((step, index) => (
           <div className="hw-connect" key={index}>
             <h3 className="hw-connect__title">{step.title}</h3>
+            <Button
+            className="hw-connect__external-btn-first"
+            type="secondary"
+            onClick={() => window.open("https://gridplus.io/?afmc=7p", "_blank")
```

Comment:
same as above. Also what does the _blank do?

Reply:
`_blank` should open in a new tab